crowbar_framework/app/views/installer/upgrades/install.html.haml

```diff
@@ -0,0 +1,39 @@
+- content_for :javascripts do
+  = javascript_include_tag "upgrader"
+
+.row
+  .col-lg-12
+    = form_tag install_upgrade_path, method: "post" do
+      .panel.panel-default
+        .panel-heading
+          %h2
+            = t("installer.upgrades.header", product: branding_config.page_title)
+
+        .panel-plain.wizard
+          = render partial: "nav", locals: { active: :install }
+
+          .wizard-body.col-lg-9
+            %h2
+              = t(".header")
+
+            %p
+              = t(".description")
+
+            - @steps.each do |step|
+              .well{step: step, style: "display:none;"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace